generators/delimited-schema/main.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+import sys
+import os
+import csv
+import time
+from unicode_csv import UnicodeCsvReader
+
+def main():
```

Comment:
Move `main` at the end of the file (and above the `if __name__ == '__main__'` block).